assets/blocks/course-outline/outline-block/outline-placeholder.test.js

```diff
@@ -30,7 +30,7 @@ describe( '<OutlinePlaceholder />', () => {
 	} );
 
 	it( 'Should render the outline placeholder correctly when feature flag is disabled', () => {
-		window.sensei.aiCourseOutline = false;
+		window.sensei.feature_flags.course_outline_ai = false;
```

Comment:
JS usually is CamelCase 🐫

Reply:
I've updated `feature_flags` to `featureFlags` in f258af564f97ff6a91bc628815c6e4f3e54d34bf. I didn't change the feature keys, which are snake_case, because it will break backward compatibility.